_P208_Nokia_LCD_5110.ino

```diff
@@ -0,0 +1,290 @@
+//#######################################################################################################
+//################################## Plugin 208: NOKIA 5110 lcd ###############################################
+//#######################################################################################################
+#include "_Plugin_Helper.h"
+#ifdef USES_P208
+
+// Working:
+// - support different digit-size's 
+// - Display Text via
+//   1.  ESPEasy-Webinterface (Line-1-Line-x)
+//   2.  http-request:
+//      - BackLight on  via httpcmd (http://ESP-IP/control?cmd=pcd8544cmd,blOn)
+//      - BackLight off via httpcmd (http://ESP-IP/control?cmd=pcd8544cmd,blOff)
+//      - Clear Display via httpcmd (http://ESP-IP/control?cmd=pcd8544cmd,clear)
+//      - Send Text via httpcmd     (http://ESP-IP/control?cmd=pcd8544,1,Hello World!)  // 1 : line#
+//        Send Text via http-request only works for the empty lines in ESPEasy-Webinterface!
+//    3. See "PLUGIN_INIT:" for pin connections.
+//
+// ToDo:
+// - different digit-size within a line....
+// - Choice of usable fonts on web-interface.
+
+#define PLUGIN_208
+#define PLUGIN_ID_208 208
+#define PLUGIN_NAME_208 "Display - LCD PCD8544 (Nokia 5110) [Test]"
+#define PLUGIN_VALUENAME1_208 "Backlight"
+#define PLUGIN_VALUENAME2_208 "Contrast"
+#define PLUGIN_VALUENAME3_208 "Rotation"
+#include <Adafruit_GFX.h>
+#include <Adafruit_PCD8544.h>
+#define lcd_lines 6
+const int digits_per_line = 48;
+
+Adafruit_PCD8544 *lcd3;
+char html_input [lcd_lines][digits_per_line];

```

Comment:
Please realize this is allocated always, even if the plugin is not used.
So you're allocating 300 bytes from the already limited amount of RAM.

Reply:
I have reduce the 300 byte's to 88 byte's by replace digits_per_line to 14.